### ISSUE TITLE
Automatic assignment not working

### ISSUE THREAD

**Issue by paidge** [pluginsGLPI/formcreator#3032]
**Describe the bug**
Since I updated GLPI v9.5 and formCreator v2.12.5 to v10.0.3 and v2.13.1, automatic assignments based on device then category seems not working when a ticket is created with a form.
Auto-assignment works when I create a ticket without the form.
My entity is well configured to do so. My devices (only applications) are also set with a manager (user or group).
The target of my form is well configured.

**To Reproduce**
1. Set your entity to assign automatically a ticket from device then category
2. Create an application with a manager
3. Create a form including a question (type GLPI Object > Applications), a title and a description
4. Add a target ticket on this form ( The ticket's answer "Associated elements" must correspond to the question)
5. Fill and submit the form
6. See that auto-assignment did not work

**Expected behavior**
The auto-assignment should be effective

**GLPI / Plugins (please complete the following information):**
- GLPI version 10.0.3
- FormCreator v2.13.2-dev
-

**Comment by paidge** [comment]
I just tried to edit the actors of the form's target in order to set the assignees with "Tech from object"/"group from object" items. But the problem is the same : auto-assignment is not effective.

**Comment by paidge** [comment]
I just reinstalled my GLPI v9 backup (with formcreator v2.12.5). The automatic assignment works when I fill my form.
Then, I updated this instance to GLPI v10.0.3 and FormCreator v2.13.1 and the automatic assignment doesn't work anymore :(

**Comment by paidge** [comment]
Auto assignements based on category works. But not on device.

**Comment by btry** [comment]
Please be patient, I'm busy for now and I cannot handle this issue fast. Remember that community support is at best effort.

**Comment by paidge** [comment]
Yes. I'm aware of that. I just give some clues for diggind the problem ;) Sorry if it seems like spam.

**Comment by btry** [comment]
No problem, they may be useful to find the problem faster.

**Comment by paidge** [comment]
After some other tests, I noticed that the auto-assignment works well for technical groups. The problem concerns only the technical manager. So I updated the issue.

**Comment by btry** [comment]
Hi

I compared Formcreator ans GLPI accrosse 9.5 and 10. It looks like I can reproduce the regression also when creating a ticket in GLPI 10 without Formcreator . Can you check if you reproduce as well ?

**Comment by paidge** [comment]
I said : 

>  Auto-assignment for technical manager works when I create a ticket without formCreator.

I 'll try again next monday for the feedback

**Comment by btry** [comment]
I'll double check on my side if I missed something.

**Comment by Ez3kield** [comment]
Hi,
I have the same issue, the auto assignement only works when i create a ticket without FormCreator.

GLPI version 10.0.5
FormCreator v2.13.4

**Comment by Ez3kield** [comment]
Hi,
The issue is still present in version 2.13.5. 
Any progress on finding the cause?
